app/src/main/java/swati4star/createpdf/util/Constants.java

```diff
@@ -68,4 +68,6 @@
 
     public static final String VERSION_NAME = "VERSION_NAME";
 
+    public static final String stateBottomSheet = "state_of_bottomsheet";
```

Comment:
use all caps for naming constant variables.

Reply:
okay